### PR TITLE
Thai translation

### PR DESCRIPTION
**Pull request by NantipatSoftEn**
some keywords and topics use in the English language are better for a clear explanation.

What should I do next? find someone for a recheck?

### CODE REVIEW

**Comment by denysdovhan** [comment]
Yes, please, find someone to check your translation.

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The original idea for WTFJS belongs to [Brian Leroux](https://twitter.com/brianl
 You can install this handbook using `npm`. Just run:
 
 ```
-$ npm install -g wtfjs
```
Comment (denysdovhan) [inline README.md:21]: Please, do not change anything in the main README, except for adding a link to your translation. Let's not fight over formatting preferences.

**Review by denysdovhan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NantipatSoftEn) [reply README.md:21]: That my mistake. so I should change back on my repository (**NantipatSoftEn/wtfjs**) right? 

This is my first pull request to contribute, Can you suggest to me plz what to do?

Also,  about find someone to check your translation. that means need to invite someone to this repository right?

Thank you

**Review by denysdovhan — COMMENTED** [review]
Please, update your branch to the latest `master`. Find a native speaker to read and verify your translation.

**Comment by NantipatSoftEn** [comment]
> Please, update your branch to the latest `master`. Find a native speaker to read and verify your translation.

 I Will do and still find someone a native speaker to read and verify

**Comment by denysdovhan** [comment]
Any updated?

**Comment by NantipatSoftEn** [comment]
> Any updated?

To be honest, I'm still can't find it, you can close this PR if you would like to do.

**Pull request closed without merging**